docs/source/algorithms/CrossCorrelate-v1.rst

```diff
@@ -9,20 +9,37 @@
 Description
 -----------
 
-Compute the cross correlation function for a range of spectra with
-respect to a reference spectrum.
+Compute the cross correlation function for a range of spectra with respect to a reference spectrum.
 
-.. math:: c_{fg}(r) = \frac{1}{\sigma_f \sigma_g} \int (f(x) - \mu_f) (g(x+r)-\mu_g) dx
+This is use in powder diffraction experiments when trying to estimate the offset (in number of bin indices) of one spectra with respect to another one.
+The input spectra should be in d-spacing and then interpolate on the X-axis of the reference before calculating the cross correlation in bin index.
+The cross correlation function is computed
 
-This is use in powder diffraction experiments when trying to estimate
-the offset of one spectra with respect to another one. The spectra are
-converted in d-spacing and then interpolate on the X-axis of the
-reference. The cross correlation function is computed in the range
-[-N/2,N/2] where N is the number of points.
+.. math:: c_{fg}(r) = \frac{1}{\sigma_f \sigma_g} \int_{XMin}^{XMax} (f(x) - \mu_f) (g(x+r)-\mu_g) dx
+
+where :math:`x` and :math:`r` are bin indices.
+:math:`r` being either half of the number of points in the requested range, ``XMin`` to ``XMax``, of the reference spectrum, or up to `MaxDSpaceShift`, whichever is smaller.
```

Comment:
since `r` is in bins and `MaxDSpaceShift` in d-spacing, I think a couple of words extra are needed here: